List-1/has23.py

```diff
@@ -0,0 +1,5 @@
+def has23(nums):
+    if nums[0]==2 or nums[0]==3 or nums[1]==2 or nums[1]==3:
+        return True
+    else:
+        return False 
```

Comment:
shouldn't hard code `0, 1` like this. What if the array length is not 2? What if you have 100 elements? 

Will you write if conditions for from `nums[0]...nums[99]`